src/i18n/nl.json

```diff
@@ -312,7 +312,7 @@
         "phone_error": "Geen geldig telefoonnummer",
         "title": "Contact",
         "url": "Website",
-        "url_error": "Geen geldige url"
+        "url_error": "Geen geldige url. Urls moeten starten met http(s)://"
```

Comment:
Should probably also be added in french and german?

Reply:
Yep indeed

Reply:
[Add other languages](https://github.com/cultuurnet/udb3-frontend/pull/920/commits/2f51f476703009632775fc46a4ee3d7b27176abe)